<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTextEdit" name="textEdit"/>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_2"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionColor"/>
    <addaction name="actionFont"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Image</string>
    </property>
    <addaction name="actionImporter"/>
    <addaction name="actionRedimensioner"/>
    <addaction name="actionD_placer"/>
    <addaction name="actionSuprimmer"/>
   </widget>
   <widget class="QMenu" name="menuHTML">
    <property name="title">
     <string>HTML</string>
    </property>
    <addaction name="actionRendu"/>
    <addaction name="actionExporter"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
   <addaction name="menuImage"/>
   <addaction name="menuHTML"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_as"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
   <addaction name="actionColor"/>
   <addaction name="actionFont"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/image/images/new.png</normaloff>:/image/images/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/image/images/open.png</normaloff>:/image/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/image/images/save.png</normaloff>:/image/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/image/images/save_as.png</normaloff>:/image/images/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/image/images/exit.png</normaloff>:/image/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/image/images/pencil.png</normaloff>:/image/images/pencil.png</iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/image/images/font.png</normaloff>:/image/images/font.png</iconset>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/image/images/copy.png</normaloff>:/image/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/image/images/cut.png</normaloff>:/image/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/image/images/paste.png</normaloff>:/image/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/image/images/edit_undo.png</normaloff>:/image/images/edit_undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/image/images/edit_redo.png</normaloff>:/image/images/edit_redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/image/images/info.png</normaloff>:/image/images/info.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionImporter">
   <property name="text">
    <string>Importer</string>
   </property>
  </action>
  <action name="actionRedimensioner">
   <property name="text">
    <string>Redimensioner</string>
   </property>
  </action>
  <action name="actionD_placer">
   <property name="text">
    <string>DÃ©placer</string>
   </property>
  </action>
  <action name="actionSuprimmer">
   <property name="text">
    <string>Suprimmer</string>
   </property>
  </action>
  <action name="actionRendu">
   <property name="text">
    <string>Rendu</string>
   </property>
  </action>
  <action name="actionExporter">
   <property name="text">
    <string>Exporter</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Ressources.qrc"/>
 </resources>
 <connections/>
</ui>
